package.json

```diff
@@ -11,6 +11,7 @@
     "test:packages": "lerna run test",
     "test": "npm run test:lint && npm run test:packages",
     "lerna:sync": "lerna exec --bail npm install && lerna publish --yes --skip-npm --skip-git --repo-version $npm_package_version",
+    "lerna:publish": "lerna publish --yes --skip-git --repo-version $npm_package_version --dist-tag alpha",
```

Comment:
note that here we are using the alpha tag. At some point we should remember to remove that :)